### ISSUE TITLE
Create a layout for arcade games

### ISSUE THREAD

**Issue by ichbinbobby** [ichbinbobby/saikava#6]
In order to keep everything consistent I want to have a layout for the arcade games. 

Criteria:

- keep the navbar from the default layout
- game shall be played in the center of the screen
- visitors can use a disqus thread at the bottom of the page

**Comment by MarioCake** [comment]
Does every game has to have its own Page and the Arcade Page is just a overview over all the games? And you are referring to that Layout?

**Comment by ichbinbobby** [comment]
Which way is more optimized? I think each game should have it's own page for highscores, how to play, a button to start the game and a disqus thread.

If you hover over Arcade in the Navigation, a drop down should appear with all the games. If you click on arcade then you wil get routed to an overview of the games. That's what I have in mind.

**Comment by MarioCake** [comment]
Everything can be optimized. It just depends. Obviously the overview would be better like you described it. 
But still you would be able to do something nifty with a one page website with some ajax calls and stuff like that. 
I started doing it like you described.

How are you imagining the thread to be? Like a little "Shoutbox" ?

**Comment by ichbinbobby** [comment]
You can see disqus thread on many pages on the internet. https://disqus.com/

Here are a few examples. Scroll down to the bottom to see them:
- http://kissanime.ru/Anime/Kobayashi-san-Chi-no-Maid-Dragon
- http://hikarinoakariost.info/miss-kobayashis-dragon-maid-original-soundtrack-ishukan-music/

**Comment by MarioCake** [comment]
For each game should be added one Disqus Thread?

**Comment by ichbinbobby** [comment]
yes, but I will take a look into that matter, since disqus should run on my account.

**Comment by Anaskhalifa** [comment]
for disqus you can start from here  https://disqus.com/api/docs/requests/
for the second you can make container and place the game app inside it 
first one i dont understand what you want

**Comment by ichbinbobby** [comment]
Framework for Javascript games: https://phaser.io/

http://www.lessmilk.com/tutorial/flappy-bird-phaser-1

**Comment by ichbinbobby** [comment]
I need to make the canvas responsive, so it will keep its aspect ratio independent from any device.

**Comment by Anaskhalifa** [comment]
Use bootstrap framework 👍
